src/crossover.rs

```diff
@@ -34,6 +36,11 @@ impl PartialEq for Crossover {
 impl Eq for Crossover {}
 
 impl Crossover {
+    /// Returns the serialized size of the Crossover
+    pub const fn serialized_size() -> usize {
+        32 * 2 + PoseidonCipher::cipher_size_bytes()
```

Comment:
We should rename it `serialized_size` to be consistent, since it's the same method used also for `from_bytes` and `to_bytes` for `PoseidonCipher`.

I open another issue for that: https://github.com/dusk-network/Poseidon252/issues/102